docs/getting-started.md

```diff
@@ -2,31 +2,50 @@
 
 **textlint** does the following steps:
 
-1. textlint load rules, every single rule is a plugin and you can add more at runtime.
-2. textlint parse *texts* using Markdown/Text/HTML parser plugin.
+1. textlint loads rules, every single rule is a plugin and you can add more at runtime.
+2. textlint parses *texts* using Markdown/Text/HTML parser plugin.
 3. textlint uses an AST([Abstract Syntax Tree](https://en.wikipedia.org/wiki/Abstract_syntax_tree "Abstract syntax tree")) to evaluate patterns in *texts*.
-4. textlint report errors/warning if exist.
+4. textlint reports errors/warning if exist.
 
+## Installation and Usage
 
-## Installation
+NOTICE: If you run the following procedures on Windows, please use PowerShell.
+
+### Create new workspace
+
+Create `<your-workspace>` directory and `package.json` file. A [package.json](https://docs.npmjs.com/files/package.json "package.json") manages dependencies of packages that include `textlint`:
+
+```
+# Create your workspace directory and move to it.
+mkdir your-workspace
+cd your-workspace
+
+# `npm init` command creates `package.json` file.
+npm init --yes
+```
+
+### Installation of textlint
    
-You can install `textlint` using npm:
+You can install `textlint` using npm. We recommend installing `textlint` locally:
```

Comment:
Lengthy a bit, but how about this?

> We recommend installing `textlint` locally, which means installing it in the `<your-workspace>/node_modules` folder, by running `npm` command with `--save-dev` option.

I'm afraid non-developer textlint user could not connect "installing locally", "`--save-dev`" option, and "`node_modules` directory" created after the command.

Reply:
I should mention at first that thank you very much for your efforts, @kongou-ae!

Reply:
I fixed this. I used more simply sentence. Please check it whether my change matches your proposal or not.